spec/README.mediawiki

```diff
@@ -1130,30 +1152,99 @@ DEX will send a list of '''Match objects'''.
 |-
 | rate      || int    || the rate that the order matched at (as quote asset per unit base asset, atoms)
 |-
-| address   || string || the client's receiving address
+| address   || string || the counterparty's receiving address
 |-
-| timestamp || int    || UTC timestamp. order broadcast time
+| sig       || string || DEX's hex-encoded signature of the serialized notification data. serialization described below
```

Comment:
I think we should list the data structure changes in the PR and commit messages.  e.g. "Remove the `timestamp` field from the "match object" definition since ..., and add a `sig` field for the signature of the serialized notification by the server.

Reply:
The timestamp removal was actually a mistake, so I'm glad you caught it.